modules/tiled-gallery.php

```diff
@@ -30,3 +30,12 @@ function jetpack_tiled_gallery_configuration_load() {
 }
 
 jetpack_load_tiled_gallery();
+
+// Tile-gallery block definition can be found in wp-calypso repo
+jetpack_register_block( 'tiled-gallery', array(
+	'render_callback' => 'jetpack_tiled_gallery_load_assets'
+) );
+function jetpack_tiled_gallery_load_assets( $attr, $content ) {
```

Comment:
Do you think you could add a docblock here so we know what the parameters refer to?